crates/proto/src/generated/rpc_store.rs

```diff
@@ -371,6 +341,30 @@ pub struct MaybeNoteScript {
     #[prost(message, optional, tag = "1")]
     pub script: ::core::option::Option<super::note::NoteScript>,
 }
+/// Represents a block range.
+#[derive(Clone, Copy, PartialEq, ::prost::Message)]
+pub struct BlockRange {
+    /// Block number from which to start (inclusive).
+    #[prost(fixed32, tag = "1")]
+    pub block_from: u32,
+    /// Block number up to which to check (inclusive). If not specified, checks up to the latest block.
+    #[prost(fixed32, optional, tag = "2")]
+    pub block_to: ::core::option::Option<u32>,
+}
+/// Represents pagination information.
```

Comment:
nit: it is pagination across a single axis of blocks, we might have different pagination axis/multiple so we should be specific in the comment